lnd_test.go

```diff
@@ -13852,6 +13852,124 @@ func testChannelBackupRestore(net *lntest.NetworkHarness, t *harnessTest) {
 	}
 }
 
+// testTLSAutoRegeneration creates an expired TLS certificate, testing that a new TLS certificate pair is regenerated when the old pair expires
```

Comment:
nit: godoc should be wrapped to 80 chars, and some of the comments in this file should be made to be complete sentences with proper punctuation :)